service/hub/main.go

```diff
@@ -24,16 +24,22 @@ func (rs *RestartableServer) Serve(c *server.Config) {
 		Addr:    c.Server.ListenAddress,
 		Handler: as,
 	}
+
+	// start the pull request counter reset timer
+	glog.Infof("Pull request counter reset timer started")
+	go as.PullReqCounter.StartResetTimer(c.Server.PullReqCounterResetInterval)
+
```

Comment:
put the goroutine under as ,not exposed in main file

Reply:
ok